pkg/daemon/ceph/osd/volume.go

```diff
@@ -679,6 +794,24 @@ func (a *OsdAgent) initializeDevices(context *clusterd.Context, devices *DeviceO
 	return nil
 }
 
+func lvmPreReq(context *clusterd.Context, pvcBacked, lvBackedPV bool) error {
```

Comment:
spelling out func names is preferable IMO
```suggestion
func lvmPreRequisite(context *clusterd.Context, pvcBacked, lvBackedPV bool) error {
```

Reply:
thoughts?